ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/router-internet-exit.yml

```diff
@@ -0,0 +1,26 @@
+router_internet_exit:
+  policies:
+    - name: po_01
+      exit_groups:
+        - name: po_eg_01_01
+        - name: po_eg_01_02
+        - name: po_eg_01_03
+        - name: po_eg_01_04
```

Comment:
let's put these in a different order and expect CLI to reorder them

Reply:
Got your point!
Done with the changes :)